programs/relayer/src/modules/eth_to_cosmos.rs

```diff
@@ -62,8 +62,11 @@ pub struct EthToCosmosConfig {
 
 impl EthToCosmosRelayerModuleService {
     async fn new(config: EthToCosmosConfig) -> Self {
+        tracing::debug!(
+            "Creating Ethereum to Cosmos relayer module service with config: {config:?}"
+        );
```

Comment:
We should put this to all modules, or no module. I'd prefer no module, but this could also go to the builder rather than individual modules

Reply:
removed